headwindsim/SU95X
# Update main to newer A32NX core

Mostly created to free up the experimental branch for the new model and gear animation.

Fixes #[issue_no]

## Summary of Changes


- Updated A32NX core to commit [9553acd](https://github.com/flybywiresim/a32nx/tree/9553acdb21ebfa3485fa5022c38cdbcc438351cd). Changes include:
  - [ADIRU] Implemented wind speed computation from TAS/GS/HDG
  - [FMGC] Show proper transition names and final approach slope from AAU1
  - [FMGC] Don't accept blank input or / for hold distance
  - [ATSU] Fix LSK6L not returning to ATSU DATALINK page in ATC MENU
  - [HYD] Trimmable physical assemblies
  - [HYD] Simulation of the rudder mechanical assembly and yaw dampers
  - [EFB] Fix and improve the pushback system
  - [EFB] Add de-boarding button to EFB payload manager
  - [MODEL] Add scroll wheel template for switches
- Update ATC ICAO aircraft type

## Screenshots (if necessary)



## References


## Additional context



Discord username (if different from GitHub):

## Testing instructions



## How to download the PR for Testing

Every new commit to this PR will cause a new Su95X artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **HEADWIND-SU95X** download link at the bottom of the page
